ramp4-pcar4/storylines-editor
# Ensure 'copy slide' always makes new RAMP config

### Related Item(s)
Issue #454 

### Changes
- [Fix] Ensures a new RAMP config file is created every time a slide with a map panel is copied, so there aren't any shared configs between slides.

### Notes
It doesn't seem we're currently _deleting_ config files when a map panel-containing slide is deleted. Should we do that?

### Testing
Steps:
1. Go to any product (`00000000-0000-0000-0000-000000000000`).
2. Create or go to a slide with a map panel. In the RAMP editor, make a few changes from the default.
3. Go to the advanced editor and note the filename of the config.
4. Copy the slide. 
5. Open the new slide, go to the map panel, the select the advanced editor. The config filename should be different from the original slide.
6. Go back to the copied map panel. In the RAMP editor, check the items you changed in step 2. They should still be changed here, even if you didn't press 'Save changes'.
